packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/CameraPermissions.java

```diff
@@ -24,15 +24,16 @@ interface ResultCallback {
   }
 
   private static final int CAMERA_REQUEST_ID = 9796;
-  private boolean ongoing = false;
+  @VisibleForTesting boolean ongoing = false;
 
   void requestPermissions(
       Activity activity,
       PermissionsRegistry permissionsRegistry,
       boolean enableAudio,
       ResultCallback callback) {
     if (ongoing) {
-      callback.onResult("cameraPermission", "Camera permission request ongoing");
+      callback.onResult("cameraPermissionRequest", "Camera permission request ongoing.");
```

Comment:
@hellohuanlin I decided to keep this error because you can get it by instantiating the camera or requesting permissions twice at times too close together. However, the app would crash and other errors result, so I added the `return` below to avoid such.

Let me know what you think of the error code and message to make it most closely conform with the other platforms!

Reply:
Would it work if you just `return` but do not report this error dart side? 
What do you expect user to do when they have this error?

Reply:
On iOS side it's not possible because the requests happen on the main thread. So duplicate requests would queue up one after another.